syft/frameworks/torch/tensors/interpreters/additive_shared.py

```diff
@@ -566,6 +591,24 @@ def chunk(tensor_shares, *args, **kwargs):
 
         module.chunk = chunk
 
+        @overloaded.function
+        def roll(tensor_shares, shifts, **kwargs):
```

Comment:
Where is this one used?

Reply:
It's used in the maxpool_deriv SecureNN protocol